FLEDGE.md

```diff
@@ -924,6 +960,8 @@ The output of `generateBid()` contains the following fields:
     browser to select only some of the returned adComponents in order to help
     make the ad k-anonymous. See [Flexible Component Ad Selection Considering k-Anonymity](#341-flexible-component-ad-selection-considering-k-anonymity)
     for more details.
+*   selectedBuyerAndSellerReportingId: (optional) A string from the interest group's ad's `selectableBuyerAndSellerReportingId` array.  If present and jointly k-anonymous with `buyerAndSellerReportingId`, `buyerReportingId`, the interest group owner, bidding script URL, and render URL, then it will be presented to `reportWin()` and `reportResult()`.
```

Comment:
>...then it will be presented to reportWin() and reportResult().

I see it is provided to `reportResult`'s `browserSignals` but not yet for `reportWin`'s.

Reply:
Never mind. After commenting I see that reportWin's description is read as an incremental (vs reportResult) and I wasn't reading closely enough.